enhancements/olm/operator-must-gather.md

```diff
@@ -0,0 +1,137 @@
+---
+title: operator-must-gather
+authors:
+  - "@shawn-hurley"
+reviewers:
+  - "@ecordell"
+  - "@njhale"
+approvers:
+  - "@ecordell"
+  - "@njhale"
+creation-date: 2020-1-6
+last-updated: 2020-1-6
+status: implementable
+see-also:
+  - "./enhancements/must-gather.md"
+  - "./enhancements/oc/inspect.md"
+---
+
+# Operator Must Gather
+
+## Release Signoff Checklist
+
+- [x] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Open Questions [optional]
+
+## Summary
+
+Components developing [must gather images]("../oc/must-gather.md#must-gather-images") to gather all the information that they will need for debugging has become the defacto mechanism for customers to father information for a support case. When debugging installation of an operator with OLM we need to gather data from objects across the cluster. 
+
+## Motivation
+
+Today back and forth between the customer and the operator team or the OLM team to get info about an operator failing install. This the current state or each operator needs to write their own must gather to capture everything. Operator authors should have a starting point for writing their own must gathers such that gathering the OLM information for their operator should be consistent. 
+
+
+### Goals
+
+List the specific goals of the proposal. How will we know that this has succeeded?
+1. Gathering should gather exactly the operator, and relatedObjects for that operator, and the resources created by OLM.
```

Comment:
It's a little unclear to me how related objects fit in.

Will this include a change to OLM so that subscription.status contains relatedObjects? Or this is saying that the olm must-gather image will collect OLM stuff by default, and users should run `oc inspect` to collect the operator-specific data from "custom" relatedObjects?

Reply:
it was a leftover line. removing it